python/dgl/contrib/sampling/sampler.py

```diff
@@ -200,7 +195,7 @@ def __iter__(self):
 
 def NeighborSampler(g, batch_size, expand_factor, num_hops=1,
                     neighbor_type='in', node_prob=None, seed_nodes=None,
-                    shuffle=False, num_workers=1, max_subgraph_size=None,
+                    shuffle=False, num_workers=1,
                     return_seed_id=False, prefetch=False):
```

Comment:
Also I assume the note "This method currently only supports MXNet backend" outdated?